### PR TITLE
Dco pubs only

### PR DESCRIPTION
**Pull request by mrpatrickwest**
A boolean facet that defaults to checked to display only DCO Publications.

### CODE REVIEW

File: ontologies/dco-s2s.ttl [hunk 1]
```diff
@@ -189,6 +189,18 @@
     s2s:hasOutput core:LabelIdContextCountJsonArray ;
     a s2s:InputValuesInterface, owl:Thing .
 
+:DcoPubs
+    core:hasDelimiter ";"^^xsd:string ;
+    a s2s:Input, owl:Thing ;
+    rdfs:comment "A search parameter to search for DCO Publications
```
Comment (zednis) [inline ontologies/dco-s2s.ttl]: accidental newline in the comment string?

File: opensearch/services/publications/search.php [hunk 2]
```diff
@@ -52,6 +52,16 @@
...
 }
 
+// Determine if we retrieve only DCO publications
+if (@$_GET['dcoPubs'] && @$_GET['dcoPubs'] != '') {
+	$constraints["dcoPubs"] = explode(";",$_GET['dcoPubs']);
+}
+else
+{
+    $constraints["dcoPubs"] = array() ;
+    $constraints["dcoPubs"][0] = "yes" ;
```
Comment (zednis) [inline opensearch/services/publications/search.php:62]: Is this how we set the default behavior?

**Comment by zednis** [comment]
:+1: I think we should fix the newline in the comment string and then this will be good to go.

File: opensearch/services/publications/search.php (hunk 2 above)
Comment (mrpatrickwest) [inline opensearch/services/publications/search.php:62]: When the first set of requests come in dcoPubs isn't set yet, like there's no selection. So yes, I default it to yes.

**Pull request merged**